packages/note/src/css/index.js

```diff
@@ -0,0 +1,129 @@
+import core from '@pluralsight/ps-design-system-core'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
+
+export const resetButton = {
+  background: 'transparent',
+  border: 'none',
+  color: 'inherit',
+  font: 'inherit',
+  lineHeight: 'normal',
+  margin: 0,
+  overflow: 'visible',
+  padding: 0,
+  width: 'auto',
+
+  MozOsxFontSmoothing: 'inherit',
+  WebkitAppearance: 'none',
+  WebkitFontSmoothing: 'inherit',
+
+  '&::-moz-focus-inner': {
```

Comment:
fancy!  Do we need all these special bits lines 15-21?  If so, how did you discover that we did?

Reply:
i pulled these styles over from prism...which probably got it from an article somewhere 😄 

i'll look at see what the current state of webkit/moz is and if there are no longer needed